e2e/features/appeal-application.feature

```diff
@@ -160,10 +168,39 @@ Feature: Complete the appeal application online
     When I select No for the Are there any new reasons your client wishes to remain in the UK or any new grounds on which they should be permitted to stay? field
     Then I click the Continue button
 
+    Given I complete the Has your client appealed against any other UK immigration decisions? page
     Given I complete the Your own reference number page
     Given I click the Save and continue button
     Then I should see an alert confirming the case has been created
 
     Given I click the Case details tab
     Then I should see No for the Are there any new reasons your client wishes to remain in the UK or any new grounds on which they should be permitted to stay? field
     Then I should not see the New matters field
+
+  @create-case @lodge-appeal @alternate
+  Scenario Outline: Lodge an appeal application (no known other appeals)
```

Comment:
It might be getting time to split out this feature file into separate alternate flows, but I can look at this as part of a separate change.